tests/functional/BaseCase.php

```diff
@@ -261,12 +393,28 @@ protected function clearGCSBucketDir(string $bucket, string $prefix): void
     {
         $client = $this->getGCSClient();
         $bucket = $client->bucket($bucket);
+        $this->log->add('DELETE OBJECTS BY PREFIX: ' . $prefix);
         $objects = $bucket->objects(['prefix' => $prefix]);
+        /** @var StorageObject $object */
         foreach ($objects as $object) {
+            $this->log->add('DELETE: ' . $object->name());
             $object->delete();
         }
     }
 
+    protected function getTestHash(): string
+    {
+        $name = $this->getName();
+        // Create a raw binary sha256 hash and base64 encode it.
+        $hash = base64_encode(hash('sha256', $name, true));
+        // Trim base64 padding characters from the end.
+        $hash = rtrim($hash, '=');
+        // remove rest of chars
+        $hash = preg_replace('/[^A-Za-z0-9 ]/', '', $hash);
+        assert($hash !== null);
+        return $hash;
+    }
```

Comment:
tady bezkolizní hash stálý pro test kejsy, který ide použit jako identifier pro všecko možné.